tests/test_auto_offer_on_device_rest.py

```diff
@@ -43,24 +53,23 @@ def on_trade(self, trade_info):
     def on_finish(self, finish_info):
         self.is_finished = True
 
-number_of_areas = 10
-load_names = [str(sys.argv[2]) if i == 0 else str(sys.argv[2])+' '+str(i+1) for i in range(number_of_areas)]
-pv_names = [str(sys.argv[3]) if i == 0 else str(sys.argv[3])+' '+str(i+1) for i in range(number_of_areas)]
-connected_devices = []
-
-for i in range(number_of_areas):
+for i in range(len(args.load_names)):
```

Comment:
How about this simpler loop over all members of  the load_names list
```suggestion
for load_name in args.load_names:
```
In general, one-latter variables (here `i`)  are not very helpful for the code readability. So please try to gice verbose names. If you really need this as a counting variable, you could have named it as `load_number` or something like this. But with my suggestion you dont need it.
Thanks!

Reply:
You're right, I'm changing it